ops/cli.py

```diff
@@ -344,28 +331,24 @@ def deploy(self, arguments) -> int:
         commands_available()
         set_up_helm()
         set_up_k8s(project=project, namespace=args.namespace, cluster=cluster, zone=args.zone)
-        with tempfile.TemporaryDirectory() as dir:
-            pwd = os.getcwd()
-            os.chdir(dir)
-            for url in ["git@github.com:broadinstitute/gotc-deploy.git",
-                        "git@github.com:broadinstitute/wfl.git"]:
-                clone(url=url, branch=args.branch)
-            tag = build("./wfl")
-            docker(directory="./wfl", tag=tag)
+
+        version = shell("cat version")
+        publish_docker_images(version=version)
+
+        pwd = os.getcwd()
+        with tempfile.TemporaryDirectory() as temp, cd(temp):
             values = "wfl-values.yaml"
-            shutil.copy(f"gotc-deploy/deploy/{gotc_folder}/helm/{values}.ctmpl", dir)
-            render_ctmpl(ctmpl_file=f"{values}.ctmpl", WFL_VERSION=tag)
+            shutil.copy(f"{pwd}/derived/2p/gotc-deploy/deploy/{gotc_folder}/helm/{values}.ctmpl", temp)
+            render_ctmpl(ctmpl_file=f"{values}.ctmpl", WFL_VERSION=version)
             helm_deploy_wfl(values=values)
 
+            container = run_cloudsql_proxy(project=project, cloudsql_instance_name="zero-postgresql")
             with open(values) as f:
                 helm_values = yaml.safe_load(f)
-            db_username = helm_values['api']['env']['ZERO_POSTGRES_USERNAME']
-            db_password = helm_values['api']['env']['ZERO_POSTGRES_PASSWORD']
-            container = run_cloudsql_proxy(project=project, cloudsql_instance_name="zero-postgresql")
-            run_liquibase_migration(db_username, db_password)
-            info("=> Stopping cloud_sql_proxy")
-            shell(f"docker stop {container}")
-            os.chdir(pwd)
+                env = helm_values['api']['env']
+                run_liquibase_migration(env['ZERO_POSTGRES_USERNAME'], env['ZERO_POSTGRES_PASSWORD'])
```

Comment:
Liquibase was using the path to the db changelog file in the cloned WFL repo so I'm not sure if this will still work 🤔  https://github.com/broadinstitute/wfl/pull/90/files#diff-da2a4da0fe1bbdf1dbae6823c2a89d66R180

Reply:
> I tried running `make` and ran into this error during the api build step. Turns out `ln -r` isn't supported on MacOS 
> 
> ```
> Writing wfl-0.1.0.jar...
> Writing target dir(s)...
> ln -sr /Users/sehsan/Devel/wfl/derived/api/target/wfl-0.1.0.jar /Users/sehsan/Devel/wfl/derived/api/target/wfl.jar
> ln: illegal option -- r
> usage: ln [-Ffhinsv] source_file [target_file]
>        ln [-Ffhinsv] source_file ... target_dir
>        link source_file target_file
> make[1]: *** [/Users/sehsan/Devel/wfl/derived/api/build.ts] Error 1
> make: *** [api] Error 2
> ```

I think you need to `brew install coreutils`

Reply:
yes you are quite right. Good spot.